src/Raven.Server/ServerWide/OperationCancelToken.cs

```diff
@@ -55,16 +60,14 @@ public void Delay()
             if (_cancelAfter == Timeout.InfiniteTimeSpan)
                 throw new InvalidOperationException("Cannot delay cancellation without timeout set.");
 
-            const int minimumDelayTime = 25;
-
-            if (_sw?.ElapsedMilliseconds < minimumDelayTime)
+            if (_sw?.ElapsedMilliseconds < _minimumDelayTimeInMs)
                 return;
 
             lock (this)
```

Comment:
Why do we even _have_ this lock? Is this cancellation token expected to be concurrent?